hw/ip/usbdev/dv/usbdev_sim_cfg.hjson

```diff
@@ -450,8 +450,6 @@
     {
       name: usbdev_rx_full
       uvm_test_seq: usbdev_rx_full_vseq
-      // No need for randomization.
-      reseed: 1
```

Comment:
Is this change intended to be part of this PR?

Reply:
Yes, as per the final comment. Perhaps you'd prefer it as a separate commit or PR? It should have been changed when that sequence was changed a couple of days ago.

Reply:
Ah indeed, sorry. Yes, I think a separate commit would be preferable. I'll leave the decision about keeping it in this PR or creating a new one to you

Reply:
Okay, separated. I'll just let it run the basic checks again, thanks.